docs/gettinginvolved/plugins.rst

```diff
@@ -0,0 +1,93 @@
+=======================
+Dataset Backend Plugins
+=======================
+
+Flocker supports pluggable storage backends; this document will teach you how to implement such a backend, with a particular focus on block device backends.
```

Comment:
:question: Not really a particular focus, that is the only backend type that this teaches you how to implement. Question mark because it does tell you how to package a plugin for any backend type. Reword if you think it can be better.